q2cli/builtin/tools.py

```diff
@@ -248,6 +250,28 @@ def formatter(*row):
                     bold=True, nl=False)
         click.echo(metadata.column_count)
 
+        if metadata.contains_renamed_columns or show_hashes:
```

Comment:
quick drive-by comment:

This doesn't let `no-show-hashes` avoid hashes if there are renamed columns.

Reply:
Oh, I didn't know we wanted it to extend that far.

Reply:
For some reason I got into my head that we wanted the default behavior to be printing the hashes whenever there were multiple files even though we had talked about not doing that before.